includes/class-wp-term-images.php

```diff
@@ -100,7 +100,7 @@ public function help_tabs() {
 	}
 
 	/**
-	 * Return the formatted output for the column row
+	 * Return the formatted output for the colomn row
```

Comment:
It was correct before, `column` 🙂

Reply:
I don't even know how that got committed. Fixed. Thanks.